pallets/phala/src/migrations/mod.rs

```diff
@@ -0,0 +1,102 @@
+/// Migrations to v4
+///
+/// From v4, all the storage version will become 4 at once. Before v4, the latest storage versions
+/// are:
+/// - fat: 1
+/// - mining: 3
+/// - mq: 0
+/// - ott: 1
+/// - registry: 1
+/// - stakepool: 1
+pub mod v4 {
+	use crate::*;
+	use frame_support::{
+		traits::{Get, StorageVersion},
+		weights::Weight,
+	};
+	use log;
+
+	type Versions = (
+		StorageVersion,
+		StorageVersion,
+		StorageVersion,
+		StorageVersion,
+		StorageVersion,
+	);
+
+	const EXPECTED_STORAGE_VERSION: Versions = (
```

Comment:
I prefer struct than tuple, struct with field name can self-describe.

I was thinking there're 6 pallets above but here's only 5 and confuse the order

Reply:
ignore me if this one not reuse in future

Reply:
Don't bother make it clear. In the future all the storage version will bump together